CMakeLists.txt

```diff
@@ -1,18 +1,21 @@
-if (CMAKE_BUILD_TYPE STREQUAL "Debug")
-    set (SSL_ROOT_PATH ${CMAKE_CURRENT_LIST_DIR}/no-asm)
-else()
-    set (SSL_ROOT_PATH ${CMAKE_CURRENT_LIST_DIR})
-endif()
+function(android_add_openssl_to_targets)
+    if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+        set (SSL_ROOT_PATH ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/no-asm)
```

Comment:
(minor comment)
As a pass by, these local variables could be turned to lowercase, it's more common naming way for cmake